content/WebAuthn/WebAuthn_Walk-Through.adoc

```diff
@@ -0,0 +1,283 @@
+== WebAuthn Walk-Through
+Build an app that enables your users to register and authenticate through WebAuthn. To implement the WebAuthn security protocol in your app, use the link:https://www.w3.org/TR/webauthn/[W3C Web Authentication API] specification and Yubico’s link:https://github.com/Yubico/java-webauthn-server/tree/master/webauthn-server-demo[Java WebAuthn Server].
+
+=== Overview
+With WebAuthn (also known as FIDO2), public-key cryptography is used to authenticate users to an online service, also known as a Relying Party (RP). When the end-user registers for an online service, an RP-specific credential key pair, i.e., a private key and a public key, is generated on the authenticator and the public key is sent to the RP (the private key never leaves the authenticator). When the user makes the request to log in, the authenticator sends an assertion that proves the user possesses the private key. The RP uses the public key to validate the assertion before allowing the user to log in.
+
+With WebAuthn also comes the concept of a resident key credential stored on the authenticator instead of encrypted and stored on the server. Resident keys enable passwordless authentication scenarios.
+
+For a detailed description of the registration and authentication ceremonies, see the “How it works” section of the link:WebAuthn_Developer_Guide/Overview.adoc[Overview] in the WebAuthn Developer Guide. In the same chapter, the sections immediately following, “FIDO2/WebAuthn application architecture” and “Key points” are also helpful.
+
+
+=== Prerequisites
+You will need at least one YubiKey, plus installation of the following:
+
+Yubico’s Java WebAuthn Server demo:: link:https://github.com/Yubico/java-webauthn-server/tree/master/webauthn-server-demo[webauthn-server-demo]
+Git:: link:https://git-scm.com/[https://git-scm.com/]
+Dependency configuration:: link:https://github.com/Yubico/java-webauthn-server#dependency-configuration[dependency-configuration]: Maven or Gradle
+Javadoc API documentation:: link:https://github.com/Yubico/java-webauthn-server/JavaDoc/webauthn-server-core/latest/com/yubico/webauthn/package-summary.html[package-summary] for the following:
+
+    * ``CredentialRepository`` interface,
+    * ``RelyingParty`` class and
+    * ``navigator.credentials.create()`` and
+    * ``navigator.credentials.get()`` methods.
```

Comment:
1. The visual formatting of this block looks off. Can you double check it locally? 
2. An unordered list seems like a path forward.
3a. In the sub unordered list, the `` are showing up on the page. Should they be removed? 
3b. Also I suggest linking to the each of the class/interface descriptions. E.g. the CredentialRepository Interface should link to https://developers.yubico.com/java-webauthn-server/JavaDoc/webauthn-server-core/latest/com/yubico/webauthn/CredentialRepository.html
3c. The navigator.credentials.create() and get() are not in the Java WebAuthn Server API documentation.  Those are in the W3C WebAuthn Spec.

Reply:
Fixed